script/card.js

```diff
@@ -0,0 +1,95 @@
+function Card() {
+    "use strict";
+    
+    var SPADES = 1;
+    var HEARTS = 2;
+    var DIAMONDS = 3;
+    var CLUBS = 4;
+    var cardWeight, suit;
+    
+    this.setCardWeight = function(n) {
+        cardWeight = n;
+    };
+    
+    this.setSuit = function(s) {
+        suit = s;
+    };
+    
+    this.getCardWeight = function() {
+        
+        return cardWeight;
+    };
+    
+    this.getSuit = function() {
+        
+        return suit;
+    };
+    
+    this.getIdCard = function() {
+        
+        return replaseNumber(cardWeight) + replaseSuit(suit);
+    };
+    
+    function replaseNumber(n) {
```

Comment:
It is better to move functions to Card.prototype as in you case they will be created for each card separately.